python/cuml/cuml/internals/input_utils.py

```diff
@@ -279,7 +279,22 @@ def determine_array_type_full(X):
     return _input_type_to_str[gen_type], gen_type in _SPARSE_TYPES
 
 
-def is_array_like(X):
+def is_array_like(X, accept_lists=False):
```

Comment:
Why this change? AFAICT this isn't called anywhere with `accept_lists=True`. If this isn't necessary I'd prefer we remove it.

Reply:
I think I might have messed up my diff, this should be called here: https://github.com/rapidsai/cuml/blob/165303958e2b2dfe13be89a77b0e2a721f8d7acb/python/cuml/cuml/internals/base.pyx#L702

Reply:
This is fixed.